source/adios2/core/IO.cpp

```diff
@@ -12,6 +12,7 @@
 #include "IO.tcc"
 
 #include <sstream>
+#include <utility>
```

Comment:
@pnorbert just a suggestion, when the C++ header has a broad meaning (e.g. algorithm or utility) I put a comment to explain why it was included. `#include <utility> //std::pair`. Helps me a lot when I have to go back and debug C++ headers. Thing like vector, map, set are self-explanatory.